torch/_dynamo/testing.py

```diff
@@ -187,7 +188,7 @@ def __init__(self):
         self.frame_count = 0
         self.op_count = 0
 
-    def __call__(self, gm: torch.fx.GraphModule, example_inputs):
+    def __call__(self, gm: torch.fx.GraphModule):
```

Comment:
Is this intentional? We aren't altering the testing behavior right?

Reply:
Right, I think we need to keep this. They should be List[torch.Tensor]